Compilers.slnf

```diff
@@ -4,16 +4,22 @@
     "projects": [
       "src\\Analyzers\\CSharp\\Analyzers\\CSharpAnalyzers.shproj",
       "src\\Analyzers\\CSharp\\CodeFixes\\CSharpCodeFixes.shproj",
+      "src\\Analyzers\\CSharp\\Tests\\CSharpAnalyzers.UnitTests.shproj",
       "src\\Analyzers\\Core\\Analyzers\\Analyzers.shproj",
       "src\\Analyzers\\Core\\CodeFixes\\CodeFixes.shproj",
       "src\\Analyzers\\VisualBasic\\Analyzers\\VisualBasicAnalyzers.shproj",
       "src\\Analyzers\\VisualBasic\\CodeFixes\\VisualBasicCodeFixes.shproj",
+      "src\\Analyzers\\VisualBasic\\Tests\\VisualBasicAnalyzers.UnitTests.shproj",
       "src\\CodeStyle\\CSharp\\Analyzers\\Microsoft.CodeAnalysis.CSharp.CodeStyle.csproj",
       "src\\CodeStyle\\CSharp\\CodeFixes\\Microsoft.CodeAnalysis.CSharp.CodeStyle.Fixes.csproj",
+      "src\\CodeStyle\\CSharp\\Tests\\Microsoft.CodeAnalysis.CSharp.CodeStyle.UnitTests.csproj",
       "src\\CodeStyle\\Core\\Analyzers\\Microsoft.CodeAnalysis.CodeStyle.csproj",
       "src\\CodeStyle\\Core\\CodeFixes\\Microsoft.CodeAnalysis.CodeStyle.Fixes.csproj",
+      "src\\CodeStyle\\Core\\Tests\\Microsoft.CodeAnalysis.CodeStyle.LegacyTestFramework.UnitTestUtilities.csproj",
+      "src\\CodeStyle\\Core\\Tests\\Microsoft.CodeAnalysis.CodeStyle.UnitTestUtilities.csproj",
       "src\\CodeStyle\\VisualBasic\\Analyzers\\Microsoft.CodeAnalysis.VisualBasic.CodeStyle.vbproj",
       "src\\CodeStyle\\VisualBasic\\CodeFixes\\Microsoft.CodeAnalysis.VisualBasic.CodeStyle.Fixes.vbproj",
+      "src\\CodeStyle\\VisualBasic\\Tests\\Microsoft.CodeAnalysis.VisualBasic.CodeStyle.UnitTests.vbproj",
```

Comment:
@sharwell @jaredpar Why are we adding all these IDE analyzer/fixers/code style projects to Compilers.sln? I don't believe the compiler team is going to need to work on these projects as part of their regular work, and this is unnecessarily bloating the solution dedicated for core compiler development. Why can't all the IDE work happen using Roslyn.sln? I would suggest we remove all of these projects from this solution.

Reply:
@Youssef1313 Meanwhile, can you please work using Roslyn.sln and avoid any changes to Compilers.slnf?